### PR TITLE
[NOTICKET] Add configurable fallbackMediaUrl

### PR DESCRIPTION
**Pull request by tinect**

### CODE REVIEW

**Review by jochenmanz — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -138,6 +139,16 @@ Adjusts the defined matcher paths for caddy. You might want to adjust those to a
 kellerkinder.staticFilePaths = "myCustomFolder/*";
 ```
 
+### kellerkinder.fallbackMediaUrl
+This url is called when the local media does not exist. Best for CDN purposes without downloading them.
```
Comment (Wysselbie) [inline README.md:143]: Maybe like this. Is more understandable to me.
```suggestion
Proxy URL for media. Best for CDN purposes without downloading them.
```

Comment (Wysselbie) [inline README.md:143]: Maybe like this. Is more understandable to me.
```suggestion
Proxy URL for media. Best for CDN purposes without downloading them.
```

File: devenv.nix [hunk 2]
```diff
@@ -224,6 +224,12 @@ in {
       default = "/theme/* /media/* /thumbnail/* /bundles/* /css/* /fonts/* /js/* /recovery/* /sitemap/*";
       description = ''Sets the matcher paths to be "ignored" by caddy'';
     };
+
+    fallbackMediaUrl = lib.mkOption {
+      type = lib.types.str;
+      default = "";
+      description = ''This url is called when the local media does not exist. Best for CDN purposes without downloading them.'';
```
Comment (Wysselbie) [inline devenv.nix:231]:
```suggestion
      description = ''Proxy URL for media. Best for CDN purposes without downloading them.'';
```

**Review by Wysselbie — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (tinect) [reply README.md:143]: I don't see this as better wording, while this is no proxy 🤔 It's an address where the client is redirected to.

Comment (Wysselbie) [reply README.md:143]: Then how about "Fallback redirect URL for media"? 🤔

**Review by Wysselbie — APPROVED** [review]

**Pull request merged**